src/screen/addressScreen/addAddress.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
+import * as RootNavigation from '../../services/navigationServices';
 import {View, StyleSheet} from 'react-native';
 import {Header} from 'components';
 import Form from './addressForm';
 
-const AddAddress = ({navigation}) => {
+const AddAddress = () => {
   return (
     <View style={Styles.container}>
       <Header
         leftIcon={'chevron-back'}
         rightIcon={'close'}
         iconSize={28}
         onLeftIconPress={() => {
-          navigation.navigate('address');
+          RootNavigation.navigate('address');
```

Comment:
If it's going back a screen.
it should use **pop()** method from navigation
because **.navigate()** will add a new screen in the navigation stack

Reply:
Added a pop function for going back.